data/data/aws/vpc/common.tf

```diff
@@ -7,6 +7,8 @@ data "aws_availability_zones" "azs" {}
 
 // Only reference data sources which are gauranteed to exist at any time (above) in this locals{} block
 locals {
+  cluster_name = "${replace(var.cluster_name, "/[^A-Za-z0-9-]/", "-")}"
```

Comment:
Can we name this variable something else, like `lb_base_name`? Having a variable with the same name in `var` and `local` and using them both in places makes it difficult to know which one is the right one to use where.